wirepas_mesh_messaging/set_config.py

```diff
@@ -46,10 +46,12 @@ class SetConfigRequest(Request):
                 Note: app_config_data/app_config_seq/app_config_data must all be defined to change one of them
                       only relevant keys for new config has to be defined
         req_id (int): unique request id
+        time_ms_epoch(int): timestamp in ms of request generation
+
     """
 
-    def __init__(self, sink_id, new_config, req_id=None, **kwargs):
-        super(SetConfigRequest, self).__init__(req_id=req_id, **kwargs)
+    def __init__(self, sink_id, new_config, req_id=None, time_ms_epoch=None, **kwargs):
+        super(SetConfigRequest, self).__init__(req_id=req_id, time_ms_epoch=time_ms_epoch, **kwargs)
```

Comment:
do we need to overload the constructor with time_ms_epoch? We should be able to pass in the time_ms_epoch parameter without having to add this parameter in the super constructor call.